### ISSUE TITLE
[Feature Request]: Introduce any Architecture Pattern.

### ISSUE THREAD

**Issue by Vishwajith-Shettigar** [Ijaiswalshivam/Talk-In#28]
**Describe the solution you'd like**
Introduce MVVM Architecture Pattern.

**Feature problem and its solution**

Many developers face challenges in structuring their codebases effectively, leading to issues such as tightly coupled components, difficulty in testing, and maintaining code. This often results in decreased productivity and scalability of the software project. The lack of a standardized architecture pattern exacerbates these problems, making it challenging for developers to adopt best practices and maintain consistency across projects.

**Comment by Ijaiswalshivam** [comment]
@Vishwajith-Shettigar assigned

**Comment by javaman97** [comment]
@Vishwajith-Shettigar Any Progress ?

**Comment by Ijaiswalshivam** [comment]
@Vishwajith-Shettigar any update?

**Comment by javaman97** [comment]
@NitinSigilipelli Would you like to work on this issue ? as it is essential for our Android App to follow an architecture pattern to allow testability and scalability in future.

**Comment by javaman97** [comment]
@Qomfortzone You can start working on it.

**Comment by Sakshi007c** [comment]
I would like to work on the architecture pattern, @javaman97

**Comment by javaman97** [comment]
> I would like to work on the architecture pattern, @javaman97

@Qomfortzone Please confirm, have you started working on this issue?

**Comment by Qomfortzone** [comment]
Yes I have started but i will take some time as my exams are going on.

**Comment by javaman97** [comment]
@Qomfortzone Have you fixed this issue ?

**Comment by Qomfortzone** [comment]
I was unable to fix it you can assign anyone else.

**Comment by javaman97** [comment]
@Sakshi007c Go for it
